.rubocop.yml

```diff
@@ -0,0 +1,24 @@
+# based on https://github.com/18F/C2/blob/2032fcb4de80ad23d9b81885be20985e99052b35/.rubocop.yml
+Metrics/LineLength:
+  Enabled: false
+Style/CommentAnnotation:
+  Enabled: false
+Style/Documentation:
+  Enabled: false
+Style/DotPosition:
+  Enabled: false
+Style/EmptyElse:
+  Enabled: false
+Style/GuardClause:
+  Enabled: false
+Style/IfUnlessModifier:
+  # inline conditionals don't get picked up by simplecov, so preferable to avoid them
+  Enabled: false
+Style/IndentArray:
+  Enabled: false
+Style/RedundantBegin:
+  Enabled: false
+Style/SignalException:
+  Enabled: false
+Style/StringLiterals:
+  Enabled: false
```

Comment:
/me is now tempted to make an `afeld_rubocop` gem with [shared configuration](https://github.com/bbatsov/rubocop#inheriting-configuration-from-a-dependency-gem) :wink: 

Actually, maybe we should [inherit from the other repository](https://github.com/bbatsov/rubocop#inheriting-configuration-from-a-remote-url) so we don't have to maintain the list in two places:

```yaml
inherit_from:
- https://raw.githubusercontent.com/18F/concourse-compliance-testing/rubocop-config/.rubocop.yml
```

replacing `rubocop-config` with `master` once https://github.com/18F/concourse-compliance-testing/pull/21 is merged.